templates/arc/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "remix-template-arc",
+  "name": "remix-template-arc-ts",
```

It's probably a good idea to run the `patchup-example-names` script?

```suggestion
  "name": "remix-template-arc",
```